ffd_info_exchange/fafsa/static/assets/css/main.css

```diff
@@ -118,10 +135,10 @@ label {
     color: #ffffff;
     display: block;
     font-size: 14px;
-    left: 50%;
+    left: 20px;
```

Comment:
Thanks for the info; makes sense. Let's merge this for now and make a refactoring ticket for moving the styles to `ffd_info_exchange`.